src/runtime/api-channel.ts

```diff
@@ -510,7 +511,7 @@ export abstract class PECInnerPort extends APIPort {
 
   ArcLoadRecipe(@RemoteMapped arc: {}, @Direct recipe: string, @LocalMapped callback: (data: {error?: string}) => void) {}
 
-  RaiseSystemException(@Direct exception: {}, @Direct methodName: string, @Direct particleId: string) {}
+  ReportExceptionInHost(@Direct exception: PropagatedException) {}
```

Comment:
I think it's probably best to provide toLiteral/fromLiteral implementations for PropagatedException and use `@ByLiteral` instead of `@Direct` - technically `@Direct` only transfers JSON and so the thing that appears at the other end of the channel wouldn't be a PropagatedException but just an Object.